daemon/mgr/container_storage.go

```diff
@@ -640,6 +640,67 @@ func (mgr *ContainerManager) Unmount(ctx context.Context, c *Container) error {
 	return os.RemoveAll(c.MountFS)
 }
 
+// SetupWorkingDirectory setup working directory for container
+func (mgr *ContainerManager) SetupWorkingDirectory(ctx context.Context, c *Container) error {
+	if c.Config.WorkingDir == "" {
+		return nil
+	}
+
+	if c.MountFS == "" {
+		mgr.setMountFS(ctx, c)
+	}
+
+	c.Config.WorkingDir = filepath.Clean(c.Config.WorkingDir)
+
+	path := filepath.Join(c.MountFS, c.Config.WorkingDir)
```

Comment:
does `runc` change user mode of the working dir?

Reply:
@fuweid , both pouch and runc do not change owner about workdir, since we do not have id map in pouch, create it as a root user with filemode 0755 is enough.